Firmware/UI/Screens/MainPage.h

```diff
@@ -29,6 +29,8 @@ class MainPage : public IMenu
                                 &_menuButton, &_shutterButton,    &_whiteBalanceButton};
 
     Color565 _backgroundColor;
+    Icon* _text;
```

Comment:
No need to define them here at all, but i will merge for now. Good work!

Reply:
Thank you!